### PR TITLE
Update /web for Foreman 3.10 GA

### PR DESCRIPTION
**Pull request by Griffin-Sullivan**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.10/Katello 4.12
* [ ] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* We do not accept PRs for Foreman older than 3.3.

### CODE REVIEW

File: web/content/index.adoc [hunk 1]
```diff
@@ -20,21 +20,18 @@ For contribution, visit the https://github.com/theforeman/foreman-documentation[
...
 
 The future version is built in nightly.
 
 * link:/release/nightly/[Nightly]
 
-There is a release candidate available for testing.
-
-* link:/release/3.10/[Foreman 3.10 & Katello 4.12]
-
 These releases are unsupported and no longer receive updates. Users should update to a supported release.
```
Comment (maximiliankolb) [inline web/content/index.adoc:32]:
```suggestion
// There is a release candidate available for testing.

These releases are unsupported and no longer receive updates. Users should update to a supported release.
```

I cannot comment on deleted lines; but it'd be nice to keep line 32 as a comment that we can reuse for the next RC.

**Review by maximiliankolb — COMMENTED** [review]
one small comment, rest LGTM

File: web/content/index.adoc (hunk 1 above)
Comment (Griffin-Sullivan) [reply web/content/index.adoc:32]: Considered this. I'll add it back

**Review by maximiliankolb — APPROVED** [review]
LGTM

**Pull request merged**